### DISCUSSION TITLE
Another option to Whitelist Stories by Regex. Exactly opposite of Block List

### DISCUSSION THREAD

**Discussion by in-sid** [spacecowboy/Feeder#273] [Ideas]
The block list option today allows to remove stories if the regex matches the title. Another option where only selected titles are needed from an RSS feed. 

I checked negating regex but it doesn't seem to work.

**Comment by in-sid** [comment]
If there's a way to work it out please help @spacecowboy

**Reply by spacecowboy** [reply]
there is not. sql globs are not regexes
